### PR TITLE
add pretty option

### PR DESCRIPTION
**Pull request by TrySound**
I'd like to make my libs as small as it possible and displaying size in bytes would be very nice.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ Default: ''
 
 Give it a title so it's possible to distinguish the output of multiple instances logging at once.
 
+##### pretty
+
+Type: `boolean`
+Default: true
+
+Displays prettified size
```
Comment (kevva) [inline readme.md]: `Displays prettified size.`. Notice the dot.

**Comment by TrySound** [comment]
@kevva Done!

File: readme.md [hunk 2]
```diff
@@ -55,6 +55,13 @@ Default: ''
 
 Give it a title so it's possible to distinguish the output of multiple instances logging at once.
 
+##### pretty
+
+Type: `boolean`
+Default: true
+
+Displays prettified size.
```
Comment (sindresorhus) [inline readme.md]: Not really clear from the description what setting this to `false` would do.

**Comment by sindresorhus** [comment]
Needs a test.

File: readme.md (hunk 2 above)
Comment (TrySound) [inline readme.md]: Can you suggest something?

Comment (kevva) [inline readme.md]: Show an example of the prettified and un-prettified output or something.

Comment (TrySound) [inline readme.md]:
```
{pretty: true} // by default
// 1.34 kB
{pretty: false}
// 1337 B
```

Comment (TrySound) [inline readme.md]: Useful when I doing refactoring for minification.

Comment (sindresorhus) [inline readme.md]: > Displays prettified size: `1337 B` → `1.34 kB`

**Pull request closed without merging**